cogs/markov.py

```diff
@@ -0,0 +1,48 @@
+"""Commands for random text generation."""
+from helpers.corpus import Corpus
+import logging
+import re
+
+from discord.ext.commands import Cog, Bot, Context, command
+
+log = logging.getLogger('bot.' + __name__)
+
+
+class Markov(Cog):
+    """Markov shenanigans!"""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.corpus = Corpus()
```

Comment:
The new Corpus helper is great and might be used in different cogs. If it is instantiated in each cog with Corpus() then each cog class will have its own copy, taking up extra memory. If we expect that 1 running bot = 1 corpus, then it should be instantiated inside corpus.py and the instance should be imported rather than the class.

Reply:
created instance in corpus.py which is imported instead of the class